core/event/event-manager.js

```diff
@@ -1527,6 +1552,10 @@ var EventManager = exports.EventManager = Montage.specialize(/** @lends EventMan
                 ? targetEntryForEventType.delete(Event_CAPTURING_PHASE)
                 : targetEntryForEventType.delete(Event_BUBBLING_PHASE)
             }
+            if(eventType === "change" && this.isBrowser && 
```

Comment:
if(eventType === "change" && this.isBrowser && 
               (target instanceof Element || (typeof SVGElement !== "undefined" && target instanceof SVGElement))) {
                this._resizeObserver.observe(target, {box: optionsOrUseCapture?.size?.box});
            }

is called twice, so let's factor it out in a private method named _isDOMTargetChangeListener()